plone/namedfile/scaling.py

```diff
@@ -17,7 +17,12 @@
 from zope.traversing.interfaces import ITraversable, TraversalError
 from zope.publisher.interfaces import IPublishTraverse, NotFound
 from zope.app.file.file import FileChunk
-from plone.protect.interfaces import IDisableCSRFProtection
+
+try:
+    # Soft dependency to make this package work without plone.protect
+    from plone.protect.interfaces import IDisableCSRFProtection
+except ImportError:
```

Comment:
Catching ImportError is not recommended http://ploneapi.readthedocs.org/en/latest/contribute/conventions.html#about-imports (see #4)